src/dowel/csv_output.py

```diff
@@ -48,6 +49,36 @@ def record(self, data, prefix=''):
                            'Did you change key sets after your first '
                            'logger.log(TabularInput)?'.format(
                                set(self._fieldnames), set(to_csv.keys())))
+                # close existing log file
+                logFileName = self._log_file.name
```

Comment:
Please follow the convention of lowercase variable names with underscores (i.e. `log_filename`).